### PR TITLE
alpaca: Make bin directories

### PR DESCRIPTION
**Pull request by tugberk92**

### CODE REVIEW

**Comment by olantwin** [comment]
I don't understand this change. In which way does this minimise the files?

Are the directories not created automatically? In that case I would make that the commit message.

**Comment by tugberk92** [comment]
Hi Oliver,
I deleted the bin directory from the ALPACA repo. Because inside the bin dir, there were some unnecessary input files.
Now, the input files are creating and deleting by makeALPACAEvents.
So, yes, I did this change for making the folders manually.
What do you suggest as a title/message?

**Comment by olantwin** [comment]
Ah, I see. Then I would just make the commit message something like "alpaca: Make bin directories".

File: alpaca.sh [hunk 1]
```diff
@@ -11,6 +11,9 @@ env:
 ---
 #!/bin/sh
 rsync -a $SOURCEDIR/* .
+mkdir bin
+mkdir bin/evrecs
+mkdir bin/outputs
```
Comment (olantwin) [inline alpaca.sh]: You could also just do `mkdir -p bin/{evrecs,outputs}`

**Comment by olantwin** [comment]
@ThomasRuf could you please merge this?

**Pull request merged**